pymongo/message.py

```diff
@@ -267,9 +267,12 @@ def namespace(self):
 
     def use_command(self, sock_info, exhaust):
         use_find_cmd = False
-        if sock_info.max_wire_version >= 4:
-            if not exhaust:
-                use_find_cmd = True
+        if sock_info.max_wire_version >= 4 and not exhaust:
+            use_find_cmd = True
+        elif sock_info.max_wire_version >= 8 and exhaust:
```

Comment:
I think the `and exhaust` is redundant here.

Reply:
Removed.